pytorch_lightning/accelerators/accelerator_connector.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 import os
+from typing import Optional, Sequence
```

Comment:
The permissions on this file have changed

0644 -> 0755

Is that on purpose?

Reply:
Probably not, but shouldn't matter, right?

Reply:
how come? git only tracks x right?

Reply:
I didn't know git tracked it. Should we revert it back ?

Reply:
reverted, and also here for all files in master: #5929